install_config/registry/accessing_registry.adoc

```diff
@@ -344,3 +344,50 @@ $ oc adm policy add-cluster-role-to-user prometheus-scraper <username>
 ----
 
 See the link:https://prometheus.io/docs/querying/basics/[upstream Prometheus documentation] for more advanced queries and recommended visualizers.
+
+[[mirror-registry-images]]
```

Comment:
i think i would have expected to find this under https://docs.openshift.org/latest/dev_guide/managing_images.html

(https://docs.openshift.org/latest/install_config/registry/accessing_registry.html#access-pushing-and-pulling-images should probably be moved there too, but that is part of a larger refactor that i think needs to happen w/ this content)

Reply:
@bparees This has nothing to do with openshift images. This command is our clone of skopeo. You do not need to have imagestream to use this command. This is just client side general tool.

Reply:
yes, but it also has nothing to do w/ the openshift registry (you can use it to mirror content between any two registries).

At least the images section is about managing images (not necessarily imagestreams).  Plus it's something a developer, not an administrator, would be doing.

Reply:
@bparees OK. I will move this to managing_images.